modules/meta.analysis/R/approx.posterior.R

```diff
@@ -43,7 +43,9 @@ approx.posterior <- function(trait.mcmc, priors, trait.data=NULL, outdir=NULL, f
     
     dat    <- trait.mcmc[[trait]]
     vname  <- colnames(dat[[1]])
-    dat    <- as.matrix(dat)[,'beta.o']
+    if('beta.o' %in% vname) {
```

Comment:
Well, as written the code would fail anyway if `dat` doesn't have `'beta.o' %in% colnames(dat)`. So I don't think I'm adding any extra possibility of failure. 

In the PDA code, `trait.mcmc[[trait]]` is just a vector of samples. It doesn't have column names, or even multiple columns for that matter. But it represents the same thing as `trait.mcmc[[trait]]$beta.o`, so still works and gives sensible results in the rest of the function.